src/blocks.rs

```diff
@@ -0,0 +1,188 @@
+use crate::mesher::CubeFace;
+
+pub struct Block<'a> {
+    pub name: &'a str,
+    pub texture_names: [&'a str; 6],
+    pub is_solid: bool,
```

Comment:
Are `name` and `texture_names` ever supposed to be non constant `str`s? Two ideas:
 * Use a `&'static str`
 * Use an enum
 
 But if they should ever be more dynamic and not just a list of `const Block` you may want to keep it as is.

Reply:
They shouldn't be dynamic. I will update the lifetime.